tap2junit/__main__.py

```diff
@@ -1,27 +1,30 @@
+#!/usr/bin/env python3
```

Comment:
Should this perhaps be just `python`? Not sure how the setuptools wrapper would work, but since we support both py2 and py3 the os/distribution should likely choose what python is default?